WORKSPACE

```diff
@@ -238,3 +238,18 @@ http_file(
         "https://github.com/bazelbuild/bazel-toolchains/archive/44200e0c026d86c53470d107b3697a3e46469c43.tar.gz",
     ],
 )
+
+load(
+    "//container/ubuntu16_04/layers/bazel:version.bzl",
+    "BAZEL_INSTALLER_SHA256",
+    "BAZEL_VERSION",
+)
+
+http_file(
+    name = "bazel_installer",
```

Comment:
add version to name, we will likely want to have multiple versions

Reply:
Done